src/Runner.Common/Constants.cs

```diff
@@ -238,6 +238,7 @@ public static class Agent
 
                 // Set this env var to "node12" to downgrade the node version for internal functions (e.g hashfiles). This does NOT affect the version of node actions.
                 public static readonly string ForcedInternalNodeVersion = "ACTIONS_RUNNER_FORCED_INTERNAL_NODE_VERSION"; 
+                public static readonly string ForcedActionsNodeVersion = "ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION"; 
```

Comment:
If we suggest customer to delete `node12` from the runner, then this is not only for Actions, we might need to update the `NodeUtil.GetInternalNodeVersion` to also respect this.

might need to consider the collision case with `ACTIONS_RUNNER_FORCED_INTERNAL_NODE_VERSION` set to `node12`